geonode_mapstore_client/static/mapstore/ms-translations/data.de-DE.json

```diff
@@ -1502,7 +1502,7 @@
                 "file": "Datei Name",
                 "format": "Format",
                 "targetStore": "Ziel",
-                "storeType": "Art des Ziels",
+                "subtype": "Art des Ziels",
```

Comment:
@mattiagiupponi we should translate only the file inside the client/translations folder and not the one in static (the content of static is generated by the npm run compile process). These storeType is not realated to the one in GeoNode so please restore these translation files